mms/export_model.py

```diff
@@ -391,12 +392,21 @@ def export_model(model_name, model_path, service_file=None, export_file=None):
 
         if model_path.startswith('~'):
             model_path = os.path.expanduser(model_path)
-
+        # Entry point model here, this is in the maibn folder
         files = os.listdir(model_path)
         onnx_file = find_unique(files, '.onnx')
         symbol_file = find_unique(files, '-symbol.json')
         params_file = find_unique(files, '.params')
-
+        #getting relative paths, for test scheduling
+        tmp = os.getcwd()
+        os.chdir(model_path)
+        # Look in the nestsed folders for other necessary model/resource files
+        for treefile in files:
```

Comment:
Do we need 3 for loops for this?  Why not just os.walk(model_path) and for f in file_names?

Reply:
I am not getting it. The symbols, params etc are calculated before your code change. None of those are used in your code change. The question is, is it enough to do a os.walk() on "model_path" rather than every file in "files". If yes, we can remove the outer for loop and just have the inner two for loops.